build/universal/trending/english.yaml

```
@@ -1,60 +1,60 @@
-article23link: "https://www.freecodecamp.org/news/javascript-string-format-how-to-format-strings-in-js/"
-article24title: "JavaScript String to Date"
-article24link: "https://www.freecodecamp.org/news/javascript-string-to-date-date-parsing-in-js/"
-article25title: "What’s a Data Analyst Do?"
-article25link: "https://www.freecodecamp.org/news/what-does-a-data-analyst-do-data-analyst-job-description/"
-article26title: "Refresh Page in JavaScript"
-article26link: "https://www.freecodecamp.org/news/refresh-the-page-in-javascript-js-reload-window-tutorial/"
-article27title: "Remove Underline from Link"
-article27link: "https://www.freecodecamp.org/news/remove-underline-from-link-in-css/"
-article28title: "How to Clear an Array in JS"
-article28link: "https://www.freecodecamp.org/news/how-to-clear-a-javascript-array-js-empty-array/"
-article29title: "Capitalize 1st Letter in JS"
-article29link: "https://www.freecodecamp.org/news/javascript-capitalize-first-letter-of-word/"
+article0title: "JavaScript forEach()"
+article0link: "https://www.freecodecamp.org/news/javascript-foreach-js-array-for-each-example/"
+article1title: "strftime() in Python"
+article1link: "https://www.freecodecamp.org/news/strftime-in-python/"
+article2title: "Check for Null in JS"
+article2link: "https://www.freecodecamp.org/news/javascript-nullable-how-to-check-for-null-in-js/"
+article3title: "Command Line Tutorial"
+article3link: "https://www.freecodecamp.org/news/command-line-commands-cli-tutorial/"
+article4title: "Check Empty String JS"
+article4link: "https://www.freecodecamp.org/news/javascript-check-empty-string-checking-null-or-empty-in-js/"
+article5title: "Make an Arrow in HTML"
+article5link: "https://www.freecodecamp.org/news/html-arrow-symbol-unicode-for-single-and-double-arrows-left-and-right-arrows/"
+article6title: "Make a List in Python"
+article6link: "https://www.freecodecamp.org/news/how-to-make-a-list-in-python-declare-lists-in-python-example/"
+article7title: "How to Read JSON in JS"
+article7link: "https://www.freecodecamp.org/news/how-to-read-json-file-in-javascript/"
+article8title: "Python Print Same Line"
+article8link: "https://www.freecodecamp.org/news/python-print-without-newline-solved/"
+article9title: "Get Current Date in JS"
+article9link: "https://www.freecodecamp.org/news/javascript-get-current-date-todays-date-in-js/"
+article10title: "HTML Font Size"
+article10link: "https://www.freecodecamp.org/news/how-to-change-text-size-in-html/"
+article11title: "Python Lowercase"
+article11link: "https://www.freecodecamp.org/news/python-lowercase-how-to-use-the-string-lower-function/"
+article12title: "Run Python Script"
+article12link: "https://www.freecodecamp.org/news/run-python-script-how-to-execute-python-shell-commands-in-terminal/"
+article13title: "indexOf in Python"
+article13link: "https://www.freecodecamp.org/news/list-indexof-for-python-how-to-get-the-index-of-an-item-in-a-list-with-index/"
+article14title: "keyerror in Python"
```

Currently the title for this article on News uses keyerror, but it should be KeyError.

I updated the title on Ghost, and we can update the title here:

```suggestion
article14title: "KeyError in Python"
```